heroku_deploy/entrypoint.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh -l
+
+send_chat_message()
+{
+  chat_path="/chat.sh"
+
+  type=$1
+  message=$2
+
+  sh -c "$chat_path $type \"$message\""
+}
+
+abort()
+{
+    echo "...error!"
+    echo ""
+    echo ""
+
+    message="*MANAGEMENT API*: Heroku deploy. Please go to project *${GITHUB_REPOSITORY}* -> Actions to see the errors."
+    type="failed"
+    send_chat_message "$type \"$message\""
+
+    exit 1
+}
+
+trap 'abort' 0
+
+set -e
+
+main(){
+  branch=$(jq --raw-output .pull_request.head.ref ${GITHUB_EVENT_PATH})
+
+  echo "-------------------------------------"
+  echo "-------------------------------------"
+  echo "||                                 ||"
+  echo "||        Deploy to heroku         ||"
+  echo "||                                 ||"
+  echo "-------------------------------------"
+  echo "-------------------------------------"
+
+  echo ""
+  echo ""
+  echo "Deploy ${branch}..."
+
+  if [ -z "${DEPLOY_ENVIRONMENT}" ] || [ "${DEPLOY_ENVIRONMENT}" != "production" ]; then
+    echo "...not being executed on production environment"
+    echo "ERROR"
+    exit 0
+  fi
+
+  message="*MANAGEMENT API: Branch ${branch} will be deployed"
```

Comment:
This should have the same action name, in that case `HEROKU DEPLOY` instead `MANAGEMENT API` to always know the action that is failing on executing.

Reply:
solved